dist/ami/build_ami.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash -e
 
-./SCYLLA-VERSION-GEN
+./SCYLLA-VERSION-GEN > /dev/null 2>&1
```

Comment:
Why this change?

Reply:
@avikivity I thought we don't need to print version number on the beginning of build_ami.sh, but the change is not related with PR so dropped the change.
Pushed new version: https://github.com/scylladb/scylla/pull/5554/files